test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13868,6 +13980,9 @@ def insert_vnode_def(self, vdef, fname=None, additive=False, restart=True):
         :param restart: If True, restart the MoM. Default is True
         :type restart: bool
         """
+        if self.is_cpuset_mom():
+            self.server.skipTest('do not modify vnodes on cpuset mom')
```

Comment:
I think this msg should like: `Creating multiple mom is not supported on cpuset mom`